packages/framework-extra-bundle/DependencyInjection/Integration/CronJobIntegration.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Draw\Bundle\FrameworkExtraBundle\DependencyInjection\Integration;
+
+use Draw\Component\Console\Command\PurgeExecutionCommand;
+use Draw\Component\Console\Entity\Execution;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+
+class CronJobIntegration implements IntegrationInterface, PrependIntegrationInterface
+{
+    use IntegrationTrait;
+
+    public function getConfigSectionName(): string
+    {
+        return 'cron-job';
+    }
+
+    public function load(array $config, PhpFileLoader $loader, ContainerBuilder $container): void
+    {
+        $this->registerClasses(
+            $loader,
+            $namespace = 'Draw\\Component\\CronJob\\',
+            $directory = \dirname(
+                (new \ReflectionClass(PurgeExecutionCommand::class))->getFileName(),
```

Comment:
not the proper class to find the relative path but this is the idea.

You will most likely have the "main service" that will be at the root of the package so you could use that one.